src/controllers/usersController.js

```diff
@@ -0,0 +1,109 @@
+'use strict';
+
+const { StatusCodes, ReasonPhrases } = require('http-status-codes');
+const usersService = require('../services/usersService');
+
+function get(_, res) {
+  res.status(StatusCodes.OK).send(usersService.get());
+}
+
+function post(req, res) {
+  const { name } = req.body;
+
+  if (!name) {
+    res
+      .status(StatusCodes.BAD_REQUEST)
+      .send(`${ReasonPhrases.BAD_REQUEST}: name is required`);
+
+    return;
+  }
+
+  res.status(StatusCodes.CREATED).send(usersService.create(name));
```

Comment:
The `post` function directly sends the result of `usersService.create(name)`. Consider adding error handling to manage potential errors from the service.